docs/layers.md

```diff
@@ -127,25 +127,50 @@ Combination of OpenStreetMap administrative boundaries (zoom >= 8), Natural Eart
 * `maritime_boundary`: a special Mapzen calculated value loosely coupled with OpenStreetMap's maritime tag, but with spatial buffer processing for lines falling in the ocean.
 * `type`: required at zooms less than 8 coming from Natural Earth for country and state (zoom 2+) boundaries, roughly equivalent to OSM's `admin_level` values.
 
-**Boundary properties (optional):**
+#### Boundary properties (optional):
 
 * `labelrank`: from Natural Earth
 * `scalerank`: from Natural Earth
 * `osm_relation`: `true`, which can also be deduced from negative `id` values.
 
-**Boundary kind values:**
-
-* `1st Order Admin Lines`, `Admin-0 country`, `Admin-1 boundary`, `Admin-1 region boundary`, `city_wall`, `country`, `county`, `Disputed (please verify)`, `fence`, `Indefinite (please verify)`, `Indeterminant frontier`, `International boundary (verify)`, `Lease limit`, `Line of control (please verify)`, `municipality`, `Overlay limit`, `retaining_wall`, `snow_fence`, `state`.
+#### Boundary kind values:
+
+* `1st Order Admin Lines`
```

Comment:
Good catch! Let's track that with: https://github.com/mapzen/vector-datasource/issues/718.